frontend/simple/views/Mailbox.vue

```diff
@@ -208,7 +189,7 @@ export default {
           let state = await latestContractState(recipient.contractId)
           let mailbox = await backend.latestHash(state.attributes.mailbox)
           let date = new Date()
-          let message = new Events.PostMessage({sentDate: date.toString(), messageType: Events.PostMessage.TypeMessage, from: this.$store.state.loggedIn, message: this.composedMessage}, mailbox)
+          let message = new Events.HashableMailboxPostMessage({sentDate: date.toString(), messageType: Events.HashableMailboxPostMessage.TypeMessage, from: this.$store.state.loggedIn.name, message: this.composedMessage}, mailbox)
```

Comment:
This line is too long (seems over 100). Break this object up into multiple lines.